.vscode/launch.json

```diff
@@ -7,19 +7,32 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "Launch Jest",
+            "name": "Test single file",
```

Comment:
Hm, i think this stuff shouldn't be here. not sure what caused the diff

Reply:
It was commited a long time ago. I can add this file to the gitignore but figured it might be useful for people